TROUBLESHOOTING.md

```diff
@@ -1,5 +1,68 @@
 # Troubleshooting
 
+## Logging
+
+Our libraries use the Java logging API via `java.util.logging` package.
+Configuring logging level reveals various facts that help your troubleshooting,
+including:
+
+- The timing of underlying client-server communication
+- Request and response message headers
+- Verbose messages in underlying dependency libraries
+
+While there are various ways to configure the logging,
+to quickly enable verbose logging for the Google Cloud Java libraries, create
+a file `logging.properties` with the following content:
+
+```
+# run java program pointing to this properties file with the java arg
+#   -Djava.util.logging.config.file=path/to/logging.properties
+handlers=java.util.logging.ConsoleHandler
+java.util.logging.SimpleFormatter.format=%1$tF %1$tT,%1$tL %4$-8s %3$-50s - %5$s %6$s%n
+
+# --- ConsoleHandler ---
+java.util.logging.ConsoleHandler.level=ALL
+java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
+.level=INFO
+
+# Specify logging level for certain packages
+com.google.api.level=ALL
+com.google.auth.level=FINE
+io.grpc.level=FINE
+
+# Example when we want to specify storge library's level
+com.google.cloud.storage.level=INFO
```

Comment:
I'm afraid this is only useful for handwritten libraries, for pure generated ones, we have very little logs, which is something we are planning to improve as part of the observability project. Can we add something to indicate it as well or maybe remove this section?

Reply:
Added that.